test/extended/topology_manager/resourcealign.go

```diff
@@ -24,6 +24,7 @@ import (
 const (
 	networkAttachmentAnnotation string = "k8s.v1.cni.cncf.io/networks"
 	sriovInterfaceName          string = "sriov1"
+	busyboxImage                string = "busybox:1.31.0"
```

Comment:
Note that this is pulling from DockerHub without a pull secret, and thus can fail to deploy due to a rate limit.

Can `registry.redhat.io/ubi8/ubi-minimal` be used instead?

Reply:
Anything which contains ping and it is maintained is fine for this task. I'll research for alternatives.